<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addProgramDialog</class>
 <widget class="QDialog" name="addProgramDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Program</string>
  </property>
  <widget class="QLabel" name="updateprogramLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>EDIT PROGRAM</string>
   </property>
  </widget>
  <widget class="QLabel" name="updprogramnamelabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PROGRAM NAME:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="updprogramNameLine">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="updprogramCodeLine">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="updprogramcodelabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PROGRAM CODE:</string>
   </property>
  </widget>
  <widget class="QLabel" name="updcollegecodelabel2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>COLLEGE CODE:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="updprcollegecodeBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="updateProgramButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>310</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>UPDATE</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
